tests/models/test_gpu.py

```diff
@@ -171,8 +171,10 @@ def test_determine_root_gpu_device(gpus, expected_root_gpu):
     pytest.param([0], [0]),
     pytest.param([1, 3], [1, 3]),
     pytest.param((1, 3), [1, 3]),
-    pytest.param('0', [0]),
-    pytest.param('3', [3]),
+    pytest.param('0', [0]),  # TODO: in v1.5 replace this by the commented line below
+    # pytest.param('0', None),
+    pytest.param('3', [3]),  # TODO: in v1.5 replace this by the commented line below
+    # pytest.param('3', [0, 1, 2]),
     pytest.param('1, 3', [1, 3]),
     pytest.param('2,', [2]),
```

Comment:
Can you add a test for `'[2]'`?

Reply:
this is not implemented. we don't document it anywhere, do we?
is it something we should support?

Reply:
nvm, got misled by https://github.com/PyTorchLightning/pytorch-lightning/issues/6228#issuecomment-787854844